lighthouse-core/audits/byte-efficiency/offscreen-images.js

```diff
@@ -72,7 +72,8 @@ class OffscreenImages extends Audit {
     const url = URL.getDisplayName(image.src, {preserveQuery: true});
     const totalPixels = image.clientWidth * image.clientHeight;
     const visiblePixels = this.computeVisiblePixels(image.clientRect, viewportDimensions);
-    const wastedRatio = 1 - visiblePixels / totalPixels;
+    // Treat images with 0 area as if they're offscreen
```

Comment:
Yea this is interesting. https://www.chromestatus.com/features uses the ring.svg image for a loading animation. When the features are loaded, the img's container is hidden. So technically, the image is needed on page load. It's just hidden later on. See any ways around such a false positive?